### DISCUSSION TITLE
Datagrid Feature Comparison Matrix

### DISCUSSION THREAD

**Discussion by texodus** [jpmorganchase/regular-table#126] [General]
### Description of Problem:
`regular-table` offers some unique features and a simple API compared to many Data Grid libraries available.  It would be helpful to see a more granular breakdown of how `regular-table` compares.  

These features make `regular-table` unique in this regard:

* It's just a `<table>`!
* Virtualized rendering for high performance.
* `async` friendly for easy remote data models.
* Easy to style, works with vanilla CSS for `<table>`.
* Small. ![npm bundle size](https://img.shields.io/bundlephobia/min/regular-table)
* Simple API.

### Potential Solutions:
From #3, which lists many such alternatives that would be a good place to start:

curated lists of grid/table projects:
- [awesome-grid](https://github.com/FancyGrid/awesome-grid)

vanilla javascript + DOM (similar to this project):
- [Clusterize](https://github.com/NeXTs/Clusterize.js)

React-based:
- [react-table](https://github.com/tannerlinsley/react-table)
- [react-datasheet](https://github.com/nadbm/react-datasheet)
- [Griddle](https://github.com/GriddleGriddle/Griddle)
- [material-table](https://github.com/mbrn/material-table)
- [rsuite-table](https://github.com/rsuite/rsuite-table)
- frameworks:
  - [react-sortable-hoc](https://github.com/clauderic/react-sortable-hoc)
  - [react-virtualized](https://github.com/bvaughn/react-virtualized)

for Angular/Vue:
- [primeng](https://github.com/primefaces/primeng/tree/master/src/app/components/table)
- [ngx-datatable](https://github.com/swimlane/ngx-datatable) 

other:
- [jqGrid](https://github.com/tonytomov/jqGrid)
- [tabulator](https://github.com/olifolkerd/tabulator)
- [tui.grid](https://github.com/nhn/tui.grid)

**Comment by telamonian** [comment]
@texodus How are you calculating that minified size?
